lib/utils/execution-state-manager.js

```diff
@@ -177,13 +178,55 @@ class ExecutionStateManager {
     this._browserToModuleMap.set(browserId, browserModuleList);
   }
 
+  /**
+   * Add module metadata mapped by moduleName to moduleMetadata Map.
+   *
+   * @param {string} moduleName
+   * @param {number} total
```

Comment:
is this total assertion or total tests?

Reply:
it's total tests. I will add to the comment